### PR TITLE
Build and deploy to AWS

### PR DESCRIPTION
**Pull request by forabi**
* [x] Create and configure `release-manager` app in AWS EB
  * [x] Create master environment
* [x] Use shared configuration and code quality checks
* [x] Deploy a beta version manually (hollowverse/hollowverse#167)
* [x] Test locally
* [x] Add health checks
  * [x] Report unhealthy if fetching environments using AWS SDK fails
* [x] Create new IAM user with read-only access to EB environments
  * [x] Get and encrypt user secrets
* [x] Write deploy script
* [x] Enable Travis builds for this repo
  * [x] Add required secrets to Travis settings
* [x] Configure TypeScript to build to `./dist`
* [x] Write `Dockerfile`
  * [x] Test Docker image
* [x] Add README

Closes hollowverse/hollowverse#186

### CODE REVIEW

File: src/health.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  isHealthy = value;
+};
+
+const health = express();
+
+health.get('/', (_, res) => {
+  res.status(isHealthy ? 200 : 500).send();
+});
+
+export { health };
```
Comment (wholesomedev) [inline src/health.ts:15]: I assume this is for health checks on the AWS platform?

File: src/releaseManager.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  let envUrl: string | undefined;
+  if (!envName || map.get(envName) === undefined) {
+    envName = getEnvName.next().value;
+  }
+
+  // Get the URL from the routing map, falling back to first environment
+  // if the environment is defined but does not have a URL
+  envUrl = map.get(envName);
+  if (!envUrl) {
+    envName = first(Array.from(map.keys())) as string;
```
Comment (wholesomedev) [inline src/releaseManager.ts]: [This StackOverflow](https://stackoverflow.com/a/32373389/604296) has other ways of getting the first element from a JS map.

**Review by wholesomedev — DISMISSED** [review]
A couple comments. Awesome work! 👍

File: src/health.ts (hunk 1 above)
Comment (forabi) [reply src/health.ts:15]: Yes, it's similar to App Engine.

File: src/releaseManager.ts (hunk 2 above)
Comment (forabi) [reply src/releaseManager.ts]: Nice! I changed the code to use the iterator

**Pull request merged**